src/utils/traverse.js

```diff
@@ -102,6 +103,7 @@ const ORDER = {
   LogicalNotOp: ['expression'],
   LogicalOrOp: ['left', 'right'],
   MemberAccessOp: ['expression'],
+  MultiplyOp: ['left', 'right'],
```

Comment:
You're right, great history is also important for me. I just rewrote those commits into more logical units, and opened a new pull request : #44